tests/TestCase/Database/TypeTest.php

```diff
@@ -158,63 +156,6 @@ public function testClear()
     }
 
     /**
-     * Tests floats from database are converted correctly to PHP
-     *
-     * @return void
-     */
-    public function testFloatToPHP()
```

Comment:
I deleted all this code because it is duplicated in the FloatType/IntegerType tests.